### DISCUSSION TITLE
[WIP] PWA 接入

### DISCUSSION THREAD

**Discussion by Mayandev** [yihong0618/running_page#250] [Ideas]
最近工作中在做 PWA（离线缓存）相关的调研，到时候看看能否将 Running Page 接入 PWA，先记个 TODO

**Comment by shaonianche** [comment]
[gatsby-plugin-manifest](https://www.gatsbyjs.com/plugins/gatsby-plugin-manifest/) gatsby的这个插件貌似就可以

**Comment by yihong0618** [comment]
感谢～
